src/main/resources/application-postgres.yaml

```diff
@@ -54,6 +54,11 @@ funksjonsbrytere:
 logging:
   config: "classpath:logback-test.xml"
 spring:
+  cloud:
```

Comment:
Hva gjør dette?

Reply:
woops! Bra catch, den skal ikke være der! Bare noe jeg trenger for å kjøre lokalt med baks-soknad-api!